### PR TITLE
Prelease 1.1.2rc

### PR DESCRIPTION
**Pull request by SeanNaren**
## What does this PR do?

Creates pre-release for 1.1.2rc after a few crucial bug fixes.

Fixes # (issue)

## Before submitting
- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you to create a separate PR for every change.
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? 
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:  

 - [ ] Is this pull request ready for review? (if not, please submit in draft mode)
 - [ ] Check that all items from **Before submitting** are resolved
 - [ ] Make sure the title is self-explanatory and the description concisely explains the PR
 - [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified; _Bugfixes should be including in bug-fix release milestones (m.f.X) and features should be included in (m.X.b) releases._
 

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+Support number for logging with `sync_dist=True` ([#5080](https://github.com/PyTorchLightning/pytorch-lightning/pull/5080)
```
Comment (Borda) [inline CHANGELOG.md:12]:
```suggestion
- Support number for logging with `sync_dist=True` ([#5080](https://github.com/PyTorchLightning/pytorch-lightning/pull/5080)
```

Comment (Borda) [reply CHANGELOG.md:12]: and for all bellow

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,9 +20,27 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Removed
 
+Disable pl optimizer by default to temporarily fix AMP issues ([#5163](https://github.com/PyTorchLightning/pytorch-lightning/pull/5163)
```
Comment (Borda) [inline CHANGELOG.md:23]:
```suggestion
Disable pl optimizer by default to temporarily fix AMP issues ([#5163](https://github.com/PyTorchLightning/pytorch-lightning/pull/5163)
```
what is the var name? "pl optimizer" >> `pl_optimizer`

File: pytorch_lightning/__init__.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 """Root package info."""
 
-__version__ = '1.1.1'
+__version__ = '1.1.2rc'
```
Comment (Borda) [inline pytorch_lightning/__init__.py:3]:
```suggestion
__version__ = '1.1.2rc1'
```

**Review by Borda — COMMENTED** [review]
add date and version in chlog

File: CHANGELOG.md [hunk 4]
```diff
@@ -18,9 +20,27 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+Fix hanging in DDP HPC accelerators ([#5157](https://github.com/PyTorchLightning/pytorch-lightning/pull/5157)
+
+
+Fix saved filename in ModelCheckpoint if it already exists ([#4861](https://github.com/PyTorchLightning/pytorch-lightning/pull/4861)
+
+
+Fix reset TensorRunningAccum ([#5106](https://github.com/PyTorchLightning/pytorch-lightning/pull/5106)
+
 
 ## [1.1.1] - 2020-12-15
```
Comment (Borda) [inline CHANGELOG.md:45]: remove "=======" bellowL58

Comment (SeanNaren) [reply CHANGELOG.md:63]: Could you repoint to where this is? I see it lower down in the release notes

Comment (Borda) [reply CHANGELOG.md:46]: it was line 58 in the next section

**Review by Borda — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -18,9 +20,28 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+- Un-balanced logging properly supported ([#5119](https://github.com/PyTorchLightning/pytorch-lightning/pull/5119)
+
+
+- Fix hanging in DDP HPC accelerators ([#5157](https://github.com/PyTorchLightning/pytorch-lightning/pull/5157)
+
+
+- Fix saved filename in ModelCheckpoint if it already exists ([#4861](https://github.com/PyTorchLightning/pytorch-lightning/pull/4861)
+
+
+- Fix reset TensorRunningAccum ([#5106](https://github.com/PyTorchLightning/pytorch-lightning/pull/5106)
```
Comment (rohitgr7) [inline CHANGELOG.md:43]:
```suggestion
- Fix saved filename in `ModelCheckpoint` if it already exists ([#4861](https://github.com/PyTorchLightning/pytorch-lightning/pull/4861)


- Fix reset `TensorRunningAccum` ([#5106](https://github.com/PyTorchLightning/pytorch-lightning/pull/5106)
```

**Review by rohitgr7 — APPROVED** [review]

**Review by tchaton — APPROVED** [review]

**Pull request merged**